test/tree.spec.ts

```diff
@@ -761,4 +731,71 @@ describe('Tree', () => {
 
     expect(tree.foldingType).toEqual(FoldingType.Collapsed);
   });
+
+  describe('Switch Folding Type', () => {
```

Comment:
And by not using `TreeService` you don't need to touch this spec file but rather internal-tree.component.spec.ts